ts/model_service_worker.py

```diff
@@ -127,6 +128,11 @@ def load_model(self, load_model_request):
                 limit_max_image_pixels = bool(load_model_request["limitMaxImagePixels"])
 
             self.metrics_cache.model_name = model_name
+            # Backwards Compatibility with releases <=0.6.0
+            # Request ID is not set for model load requests
+            # TODO: UUID serves as a temporary request ID for model load requests
+            self.metrics_cache.set_request_ids(str(uuid.uuid4()))
+
```

Comment:
Here is just initialize metrics_cache in service, not real model load metrics. uuid should be set within func load. So far there is no model loading metrics as i know.

Reply:
Moved logic for `request id` assignment for model load requests from `model_service_worker.py` to [`model_loader.py`](https://github.com/pytorch/serve/pull/2769/files#diff-bd9cbbce71a9f862d4edafb574d20bd4dcdf5ae8f80bf2aa8ad3b73e80fb5807R93-R99).

That's correct, we don't have any default metrics that we emit during model load time, but a custom handler can emit metrics in the `initialize` method. If the request id is not set, it can cause the metric to only have `Level:Error` as the default dimension that gets added instead of `ModelName:<model_name>` and `Level:Model`. Described the issue here: https://github.com/pytorch/serve/issues/2772

Reply:
Setting uuid for model loading should be at func [load](https://github.com/pytorch/serve/blob/master/ts/model_loader.py#L67), not here.

Reply:
Agreed. I've already moved it to the [load](https://github.com/pytorch/serve/pull/2769/files#diff-bd9cbbce71a9f862d4edafb574d20bd4dcdf5ae8f80bf2aa8ad3b73e80fb5807R93-R99) method: https://github.com/pytorch/serve/pull/2769/files#diff-bd9cbbce71a9f862d4edafb574d20bd4dcdf5ae8f80bf2aa8ad3b73e80fb5807R93-R99